### PR TITLE
Add SR user and locale to /config page

### PR DESCRIPTION
**Pull request by fernandog**

### CODE REVIEW

File: gui/slick/interfaces/default/config.tmpl [hunk 1]
```diff
@@ -17,6 +18,7 @@
 	<h1 class="title">$title</h1>
 #end if
 
+##set sr_user = $getpass.getuser()
```
Comment (abeloin) [inline gui/slick/interfaces/default/config.tmpl]: This should be in a try..except as getuser can return an exeception if unable to find the username

**Comment by fernandog** [comment]
@abeloin what about how?

**Comment by fernandog** [comment]
@Hellowlol can you try this in windows?

**Comment by Hellowlol** [comment]
Worked on windows 7 home atleast. Not sure i like sr grabbing my username, im kinda paranoid...

**Comment by fernandog** [comment]
@Hellowlol we need this because of all this issues:
https://github.com/SiCKRAGETV/sickrage-issues/issues?q=is%3Aissue+codec+can%27t+encode

**Comment by abeloin** [comment]
@fernandog There is no need to try getpwuid as it will be tried by getpass.user function. You just need to enclose getpass.user with a try block catching the exception and returning "unable to determine the current user" message.

**Comment by fernandog** [comment]
@abeloin 

in myt setup getpass returns "root":
SR User:        root
it's wrong. its pi

**Comment by abeloin** [comment]
How do you start sickrage?

**Comment by fernandog** [comment]
@abeloin 


\etc\default\sickbeard
```
SB_USER=pi
SB_HOME=/home/pi/SickRage
SB_DATA=/home/pi/SickRage
RUN_AT_STARTUP="YES"
```

**Comment by fernandog** [comment]
@Hellowlol I got import error (import pwd) in my python 2.7.9 in Windows.

**Comment by Hellowlol** [comment]
yeah, that you you need to cache it like i said.

**Comment by fernandog** [comment]
@Hellowlol but I catch it, don't I

```
+#try
+#import pwd
+#set sr_user = $pwd.getpwuid(os.getuid()).pw_name
+#except ImportError
+#import getpass
```

**Comment by abeloin** [comment]
@fernandog what is your init file?

**Comment by fernandog** [comment]
@abeloin it's a modified version of ubuntu/debian

It checks If Im on ethernet and if my HDD is mounted
https://gist.github.com/fernandog/991879b11bd5f8734113

**Comment by Hellowlol** [comment]
Honestly i dont know, cheetah makes my brain hurt. Anyway try this one:

```
try:
    import pwd
    set_user = pwd.getpwuid(os.getuid()).pw_name
except ImportError:
    try:
        import getpass
        set_user = getpass.getuser()
    except:
        pass


print set_user
```

**Pull request merged**